src/modules/royalty.ts

```diff
@@ -0,0 +1,152 @@
+import { Royalty, Royalty__factory } from "@3rdweb/contracts";
+import { BigNumberish } from "ethers";
+import { ModuleType, Role, RolesMap } from "../common";
+import { Currency, getCurrencyMetadata } from "../common/currency";
+import { Module } from "../core/module";
+
+interface IRoyalty {
+  /**
+   * Get the total shares held by payees
+   */
+  totalShares(): Promise<BigNumberish>;
```

Comment:
i know this is going to be inconsistent but i think we should:
ALL input of big number type / id = BigNumberish
ALL output / result of big number = BigNumber

reason being that on the input side we'd support number, string or big number.
output = it's expected that it's always bignumber and not int / string

Reply:
this makes sense to me and I like the idea of this being the case always